src/app/search/directives/search-series.component.ts

```diff
@@ -0,0 +1,74 @@
+import { Component, Input, Output, EventEmitter } from '@angular/core';
+import { CATEGORIES, SUBCATEGORIES } from '../../shared/model/story.categories';
+
+@Component({
+  selector: 'publish-search-series',
+  styleUrls: ['search-series.component.css'],
+  template: `
+    <div class="form-group">
+      <!-- TODO: delay search until finish typing -->
+      <input (keyup)="searchByText()" [(ngModel)]="searchText" placeholder="search by title or description"/>
```

Comment:
Hmm, I think `keyup` is because at first I used the `change` event and it didn't update until the field lost focus. `ngModelChange` seems better than event binding. I made an attempt at debounce like this example https://angular.io/docs/ts/latest/guide/server-communication.html#!#more-observables, but I screwed it up and reverted. It's definitely needed though. Seems like it's gonna take some refactoring, separate PR?